langchain/src/output_parsers/expression_type_handlers/base.ts

```diff
@@ -1,104 +1,78 @@
-import type meriyahT from "meriyah";
+import { GRAMMAR } from "./grammar/parser_grammar.js";
 
 export abstract class NodeHandler {
   constructor(protected parentHandler?: NodeHandler) {}
 
-  abstract accepts(
-    node: meriyahT.ESTree.Node
-  ): Promise<meriyahT.ESTree.Node | boolean>;
+  abstract accepts(node: ExpressionNode): Promise<ExpressionNode | boolean>;
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  abstract handle(node: meriyahT.ESTree.Node): Promise<any>;
+  abstract handle(node: ExpressionNode): Promise<any>;
 }
 
 export class ASTParser {
-  static astParseInstance: typeof meriyahT.parseScript;
+  static astParseInstance: ParseFunction;
 
   static async importASTParser() {
     try {
       if (!ASTParser.astParseInstance) {
-        const meriyah = await import("meriyah");
-        ASTParser.astParseInstance =
-          meriyah.parseScript as typeof meriyahT.parseScript;
+        const { default: peggy } = await import("peggy");
+        const parser = peggy.generate(GRAMMAR);
+        const { parse } = parser;
+        ASTParser.astParseInstance = parse as ParseFunction;
       }
       return ASTParser.astParseInstance;
     } catch (e) {
-      console.log(e);
       throw new Error(
-        "Failed to import meriyah. Please install meriyah (i.e. npm install meriyah)."
+        `Failed to import peggy. Run 'npm install peggy' to install the package.`
```

Comment:
Keep style like the old comment, as people may be using `yarn`

Reply:
Ok I'll change it.